src/deriver/queue_manager.py

```
@@ -766,11 +767,19 @@ async def _cleanup_work_unit(
     logger.debug("Starting queue manager")
+    try:
+        await init_cache()
+    except Exception as e:
+        logger.warning(
+            "Error initializing cache in queue manager; proceeding without cache: %s", e
+        )
+
     manager = QueueManager()
     try:
         await manager.initialize()
     except Exception as e:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)